google_guest_agent/network/dhclient.go

```diff
@@ -0,0 +1,109 @@
+package network
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/guest-agent/google_guest_agent/run"
+	"github.com/GoogleCloudPlatform/guest-agent/metadata"
+	"github.com/GoogleCloudPlatform/guest-agent/utils"
+	"github.com/GoogleCloudPlatform/guest-logging-go/logger"
+)
+
+type dhclient struct{}
+
+func (n dhclient) IsManaging(ctx context.Context, iface string) (bool, error) {
+	// Check if the dhclient CLI exists.
+	result := run.WithOutput(ctx, "command", "-v", "dhclient")
```

Comment:
You can use `exec.LookPath`  to verify existence of that command.

Reply:
Done.